adapter/src/main/java/org/mifos/chatbot/adapter/handler/loan/ClientActivationDateHandler.java

```diff
@@ -30,7 +31,7 @@
 @Slf4j
 @Component
 public class ClientActivationDateHandler extends BaseLoanIntentHandler {
-    private static final String[] INTENT_KEYWORDS = {"client", "activation", "date"};
+    private static final String[] INTENT_KEYWORDS = {"client_activation_date"};
```

Comment:
All such hard coded strings to static files

Reply:
same as above.

Reply:
Resolved